controllers/index.js

```diff
@@ -2,9 +2,13 @@ const express = require('express');
 const router = express.Router();
 const userRoute = require('./users');
 const socialMediaRoute = require('./socialmedia');
+const photoRoute = require('./poto');
+const commentRoute = require('./comment')
 const verifyToken = require('../middleware/verifyToken');
 
 router.use('/users', userRoute);
 router.use('/socialmedias', verifyToken, socialMediaRoute);
+router.use('/photo', verifyToken,photoRoute);
+router.use('/comment', verifyToken,commentRoute);
```

Comment:
kalo di kode.id harusnya `/photos` & `/comments`, diubah yaa
```suggestion
router.use('/photos', verifyToken,photoRoute);
router.use('/comments', verifyToken,commentRoute);
```